src/components/NavigationBar.js

```diff
@@ -1,240 +1,259 @@
-
-
-import {useState} from "react";
+import { useState } from "react";
 import { useLocation, useNavigate } from "react-router-dom";
 import { useAuthState } from 'react-firebase-hooks/auth';
 import { auth } from '../firebase';
 import { handleLogout } from '../functions/Auth';
 import SideBar,{SideBarItem} from './SideBar.js';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { faHome, faUser, faPlus, faSignOutAlt, faSignInAlt, faPersonCircleQuestion, faBookBookmark, faGear, faBell } from '@fortawesome/free-solid-svg-icons';
+import { faHome, faUser, faPlus, faSignOutAlt, faSignInAlt, faPersonCircleQuestion, faBookBookmark, faGear, faBell, faBars } from '@fortawesome/free-solid-svg-icons';
 import { Link } from 'react-router-dom';
 import Searchbar from './shared/Searchbar.js';
 import { handleSearch } from '../functions/searchFunctions';
-import { Tooltip } from 'react-tooltip';
 import { onSnapshot } from 'firebase/firestore';
 import { db } from '../firebase';
 import { useEffect } from 'react';
 import { collection } from 'firebase/firestore';
+import { useMediaQuery } from 'react-responsive';
 
-
-import "../styles/sideBar.css"
 export default function NavigationBar() {
   const [user, setUser] = useState(null);
   const [searchResults, setSearchResults] = useState([]);
   const [searchStatus, setSearchStatus] = useState('');
   const [newNotifications, setNewNotifications] = useState(false);
   const location = useLocation();
   const navigate = useNavigate();
-  // Fetch notifications from the database
```

Comment:
Can you not delete these comments? They explain the functions below. Also, can you write few comments explaining what your code does?

Reply:
added the comments back